From b739dd6bcc92e6aeff1d8c1e12a2bda7479eff1d Mon Sep 17 00:00:00 2001
From: Martin Wilck <martin_wilck@gmx.de>
Date: Sun, 27 Mar 2022 23:14:14 +0200
Subject: [PATCH] add local dependencies mirror

---
 build.gradle         | 11 +++++++----
 schemas/build.gradle | 13 ++++++++-----
 2 files changed, 15 insertions(+), 9 deletions(-)

diff --git a/build.gradle b/build.gradle
index 7dca6b8..1946a50 100644
--- a/build.gradle
+++ b/build.gradle
@@ -64,12 +64,15 @@ run {
 }
 
 repositories {
-    jcenter()
     maven {
-        name = 'JBoss repository' // required to obtain non-free JAI
-        url = 'https://repository.jboss.org/nexus/content/repositories/thirdparty-releases'
+        url 'dependencies'
     }
-    mavenLocal()
+    jcenter()
+    // maven {
+       // name = 'JBoss repository' // required to obtain non-free JAI
+        // url = 'https://repository.jboss.org/nexus/content/repositories/thirdparty-releases'
+    // }
+    // mavenLocal()
     //flatDir(dirs: 'dev/externals') // for libraries not in any other repository
 }
 
-- 
2.34.1

