commit 36edb484acfe1d5dee59d017f8f05d4413b1b21f
Author: Martin Wilck <martin_wilck@gmx.de>
Date:   Sun Apr 3 22:38:28 2022 +0200

    audiveris 5.2.5: patch for offline building

diff --git a/build.gradle b/build.gradle
index 075f494cf..34d4f6e2d 100644
--- a/build.gradle
+++ b/build.gradle
@@ -62,8 +62,7 @@ run {
 
 repositories {
     maven {
-        name = 'JBoss repository' // required to obtain non-free JAI
-        url = 'https://repository.jboss.org/nexus/content/repositories/thirdparty-releases'
+        url 'dependencies'
     }
     mavenLocal()
     mavenCentral()
diff --git a/schemas/build.gradle b/schemas/build.gradle
index a4fb63aa8..c3c116da1 100644
--- a/schemas/build.gradle
+++ b/schemas/build.gradle
@@ -48,11 +48,10 @@
 apply plugin: 'java'
 
 repositories { 
-    mavenCentral() 
     maven {
-        name = 'mvnrepository' 
-        url = 'https://mvnrepository.com/artifact/net.sf.saxon/Saxon-HE'
+        url '../dependencies'
     }
+    mavenCentral() 
 }
 
 dependencies { 
@@ -316,4 +315,4 @@ task checkXsltProc {
         
         ///genHtmls.dependsOn("newGenHtml$cls")    
     }
-}
\ No newline at end of file
+}
